### ISSUE TITLE
in POST v2/subscriptions, with "geometry" condition expression and without "coords" field

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2406]
#### Request
```
url: POST http://localhost:1026/v2/subscriptions
headers:
    Content-Type: application/json
    Fiware-Service: test_condition_expression_geometry_error
    Fiware-ServicePath: /test
payload:{"notification": {"http": {"url": "http://localhost:1234"}, "attrs": ["temperature"]}, "expires": "2016-04-05T14:00:00.00Z", "subject": {"entities": [{"idPattern": ".*", "type": "room"}], "condition": {"expression": {"georel": "disjoint"}, "attrs": ["temperature"]}}}
```
#### Response 
```
http code: 201 - Created
headers:
   date: Fri, 29 Jul 2016 12:54:45 GMT
   fiware-correlator: a0058e58-558b-11e6-a249-005056a20feb
   connection: Keep-Alive
   content-length: 0
   location: /v2/subscriptions/579b5215c80eb79a267b4aa5
payload:
```
#### Expected
```
http code: 400 - Bad Request
payload: {"error":"BadRequest","description":"error parsing geo-query fields: error parsing georel: geometry does not exist"}
```
#### Request
```
url: POST http://localhost:1026/v2/subscriptions
headers:
    Content-Type: application/json
    Fiware-Service: test_condition_expression_geometry_error
    Fiware-ServicePath: /test
payload: {"notification": {"http": {"url": "http://localhost:1234"}, "attrs": ["temperature"]}, "expires": "2016-04-05T14:00:00.00Z", "subject": {"entities": [{"idPattern": ".*", "type": "room"}], "condition": {"expression": {"geometry": "point"}, "attrs": ["temperature"]}}}
```
#### Response 
```
http code: 201 - Created
headers:
   date: Fri, 29 Jul 2016 12:54:45 GMT
   fiware-correlator: a0058e58-558b-11e6-a249-005056a20feb
   connection: Keep-Alive
   content-length: 0
   location: /v2/subscriptions/579b5215c80eb79a267b4aa5
payload:
```
#### Expected
```
http code: 400 - Bad Request
payload: {"error":"BadRequest","description":"error parsing geo-query fields: error parsing geometry: coords does not exist"}
```
#### Request
```
url: POST http://localhost:1026/v2/subscriptions
headers:
    Content-Type: application/json
    Fiware-Service: test_condition_expression_geometry_error
    Fiware-ServicePath: /test
payload: {"notification": {"http": {"url": "http://localhost:1234"}, "attrs": ["temperature"]}, "expires": "2016-04-05T14:00:00.00Z", "subject": {"entities": [{"idPattern": ".*", "type": "room"}], "condition": {"expression": {"coords": "25.774,-80.190"}, "attrs": ["temperature"]}}}
```
#### Response 
```
http code: 201 - Created
headers:
   date: Fri, 29 Jul 2016 12:54:45 GMT
   fiware-correlator: a0058e58-558b-11e6-a249-005056a20feb
   connection: Keep-Alive
   content-length: 0
   location: /v2/subscriptions/579b5215c80eb79a267b4aa5
payload:
```
#### Expected
```
http code: 400 - Bad Request
payload: {"error":"BadRequest","description":"error parsing geo-query fields: error parsing coords: geometry does not exist"}
```

**Comment by fgalan** [comment]
Fixed by PR #2411

**Comment by iariasleon** [comment]
LGTM
```
http code: 400 - Bad Request
headers:
   date: Mon, 01 Aug 2016 12:37:53 GMT
   fiware-correlator: c3d482c0-57e4-11e6-8510-005056a20feb
   connection: Keep-Alive
   content-type: application/json
   content-length: 119
payload: {"error":"BadRequest","description":"partial geo expression; geometry, georel and coords have to be provided together"}
```
